jerry-core/jcontext/jcontext.h

```diff
@@ -56,9 +72,17 @@ typedef struct jerry_context_data_header
  * The purpose of this header is storing
  * all global variables for Jerry
  */
-typedef struct
+struct jerry_context_t
```

Comment:
Why do you remove the typedef?

Reply:
It is typedef'd in jerryscript-core.h. A typedef cannot be declared twice. `jerry_context_t` is taking over the place of `jerry_instance_t`, so it has to be opaquely forward declared in the public API header. But that means that only the struct can be declared here.

Note: `jerry_instance_t` was handled exactly the same way.